src/celeritas/field/FieldDriver.hh

```diff
@@ -274,7 +274,7 @@ CELER_FUNCTION DriverResult FieldDriver<StepperT>::accurate_advance(
     // accumulation
     CELER_ENSURE(curve_length > 0
                  && (curve_length <= step || soft_equal(curve_length, step)));
-    output.end.step = curve_length;
+    output.end.step = min(curve_length, step);
```

Comment:
Nothing to do with this MR, but I find that both `min` and `celeritas::min` are used in our codes.